src/main/resources/config/application-rsocket.yml

```
@@ -0,0 +1,4 @@
+spring.rsocket.server:
+  transport: websocket
+  mapping-path: /rsocket
+  port: 7000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)